javascript/ql/test/query-tests/Security/CWE-022/TaintedPath/examples/TaintedPath.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs'),
+      http = require('http'),
+      url = require('url');
+
+const ROOT = "/var/www/";
+
+var server = http.createServer(function(req, res) {
+  let filePath = url.parse(req.url, true).query.path;
+
+  // BAD: This could read any file on the file system
```

Comment:
```suggestion
  // BAD: This function uses unsanitized input that can read any file on the file system.
```

Small suggestion for extra context/ease of localization